### PR TITLE
added sources into snowflake quickstart guide

### PR DESCRIPTION
**Pull request by jeremyholtzman**
## What are you changing in this pull request and why?

- I added a section for `sources` to the Snowflake quickstart guide. I think it would be super beneficial to include sources in these quickstart guides so we can encourage folks to follow our best practices and explicitly define their sources instead of using the hard-coded table reference.
- I also switched the "Deploy" section to use the `dbt build` command instead of `dbt run` and `dbt test` to follow our best practices

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

**Comment by MichelleBaird-DBT** [comment]
Looks good to me! This PR has approval from a training perspective. Waiting to get approval for technical accuracy @nghi-ly  ,then it should be good to go!

File: website/docs/quickstarts/snowflake-qs.md [hunk 1]
```diff
@@ -397,6 +398,73 @@ Later, you can connect your business intelligence (BI) tools to these views and
 <FAQ src="Models/unique-model-names" />
 <FAQ src="Project/structure-a-project" alt_header="As I create more models, how should I keep my project organized? What should I name my models?" />
 
+## Build models on top of sources
+
+Sources make it possible to name and describe the data loaded into your warehouse by your Extract and Load tools. By declaring these tables as sources in dbt, you can then
```
Comment (nghi-ly) [inline website/docs/quickstarts/snowflake-qs.md]:
```suggestion
Sources make it possible to name and describe the data loaded into your warehouse by your extract and load tools. By declaring these tables as sources in dbt, you can:
```

File: website/docs/quickstarts/snowflake-qs.md [hunk 2]
```diff
@@ -397,6 +398,73 @@ Later, you can connect your business intelligence (BI) tools to these views and
...
+- select from source tables in your models using the `{{ source() }}` function, helping define the lineage of your data
+- test your assumptions about your source data
+- calculate the freshness of your source data
+
+1. Create a new YML file `models/sources.yml`.
+2. In order to declare the the sources, copy the following into the file and click **Save**.
```
Comment (nghi-ly) [inline website/docs/quickstarts/snowflake-qs.md]:
```suggestion
2. Declare the sources by copying the following into the file and clicking **Save**.
```

File: website/docs/quickstarts/snowflake-qs.md [hunk 3]
```diff
@@ -397,6 +398,73 @@ Later, you can connect your business intelligence (BI) tools to these views and
...
+        status
+
+    from {{ source('jaffle_shop', 'orders') }}
+    ```
+
+    </File>
+
+5. Execute `dbt run`. 
+
+    The results of your `dbt run` will be the exaxt same as the previous step. Your `stg_cusutomers` and `stg_orders`
```
Comment (nghi-ly) [inline website/docs/quickstarts/snowflake-qs.md]:
```suggestion
    The results of your `dbt run` will be exactly the same as the previous step. Your `stg_cusutomers` and `stg_orders`
```

File: website/docs/quickstarts/snowflake-qs.md [hunk 4]
```diff
@@ -397,6 +398,73 @@ Later, you can connect your business intelligence (BI) tools to these views and
...
+    models will still query from the same raw data source in Snowflake. By using `source`, you get the added benefits
+    of being able to tests and document your raw data, as well as understand the lineage of your sources.
```
Comment (nghi-ly) [inline website/docs/quickstarts/snowflake-qs.md]:
```suggestion
    test and document your raw data and also understand the lineage of your sources. 
```

File: website/docs/quickstarts/snowflake-qs.md [hunk 5]
```diff
@@ -397,6 +398,73 @@ Later, you can connect your business intelligence (BI) tools to these views and
...
+
+    from {{ source('jaffle_shop', 'orders') }}
+    ```
+
+    </File>
+
+5. Execute `dbt run`. 
+
+    The results of your `dbt run` will be the exaxt same as the previous step. Your `stg_cusutomers` and `stg_orders`
+    models will still query from the same raw data source in Snowflake. By using `source`, you get the added benefits
```
Comment (nghi-ly) [inline website/docs/quickstarts/snowflake-qs.md]:
```suggestion
    models will still query from the same raw data source in Snowflake. By using `source`, you can

**Review by nghi-ly — COMMENTED** [review]
looks great, @jeremyholtzman , thank you!

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**